.github/workflows/run_test_cases.yml

```diff
@@ -12,71 +12,76 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.actor }}-${{ github.head_ref || github.run_id }}
   cancel-in-progress: true
 
+env:
+  PR_NUMBER: ${{ github.event.issue.number || github.event.pull_request.number }}
+
 jobs:
   Win-RunTestCases:
     if: |
       contains(github.event.pull_request.body, '[X] needs automatic test cases check') ||
-      contains(github.event.comment.body, '@cocos-robot run test cases')
+      (
+        contains(github.event.comment.body, '@cocos-robot run test cases') &&
+        contains(fromJson('["OWNER", "MEMBER"]'), github.event.comment.author_association)
```

Comment:
@RedYetiDev Why do we need to add this line `contains(fromJson('["OWNER", "MEMBER"]'), github.event.comment.author_association)`? Any reasons? 

This change causes the workflow being skipped.

And why didn't you also update which in macOS-RunTestCases job?

Reply:
That line prevents a non-member from triggering the workflow.

Reply:
I'm the member, but the workflow will be skipped.I dont know why.

Reply:
Strange, I'll look into it